vcd/validate_funcs.go

```diff
@@ -171,6 +171,31 @@ func IsIntAndAtLeast(min int) schema.SchemaValidateFunc {
 	}
 }
 
+// emptyOrIsIntAndAtLeast returns a SchemaValidateFunc which tests if the provided value is empty or
+// its string value convertable to int and is at least min (inclusive)
+func emptyOrIsIntAndAtLeast(min int) schema.SchemaValidateFunc {
```

Comment:
Just a note (not asking for a change), but the other day I discovered one can combine several `validation` functions in schema:

```
ValidateDiagFunc: validation.AnyDiag(
	validation.ToDiagFunc(validation.StringIsEmpty),
	validation.ToDiagFunc(validation.IntAtLeast(1))),
```

or

```
ValidateFunc: validation.Any(
	validation.StringIsEmpty,
	validation.IntAtLeast(1)),
```

Reply:
Thanks. That is a good approach. I am using it now, just had to use our own `IsIntAndAtLeast` instead of `validation.IntAtLeast(1)` (because `validation.IntAtLeast(1)` requires `TypeInt`, but we're using `TypeString` to handle 0 vs unset here)

Test passed.